### ISSUE TITLE
R package for storage/mapping of Australian-specific data sets

### ISSUE THREAD

**Issue by jonocarroll** [ropensci/auunconf#23]
_(I've done nothing in the way of seeing if anything like this already exists, so this may be a non-starter)._

In a similar-yet-different approach to a [previous entry](https://github.com/ropensci/geojsonio), and which could tie in with #20, #16, and #6, what about suring up some Aussie maps with appropriate meta-data? States/boundaries/local boundaries/major and minor cities/etc... 

I know that `ggmap::get_map` can get the ball rolling on a location, but I'd like to see (if it doesn't already exist) a package that loads up a heap of up-to-date and historical Australian data sets that can be added to a stored shapefile map, with the appropriate level factors to do so by state/city/etc... possibly without having to refer to an external map source.

```
plot_map_AUS("average_rainfall", "Adelaide", 2014)
```

If one was doing an analysis on such data, that could be extracted and processed however one liked, 

```
histRainData_Adelaide <- plot_map_AUS("average_rainfall", "Adelaide", 2004:2014, plot=FALSE)
```

No more going and getting the ll/ul/lr/ur from the map bounding box, no more manually looking up a lat/lon to go get a map, or having to wait for the Google API limit to reset. Plot the Australian data NOW.

**Comment by jonocarroll** [comment]
I'd also love to see a new `ggplot2` `geom` or `theme` that plots either Australia as a whole (incl. Tas) or as an exploded view.

[The first of those seems well-covered for the USA](https://github.com/hrbrmstr/albersusa). The latter, well, that's left as an interesting option.

**Comment by Stephen-Gates** [comment]
Here's [the data](http://data.gov.au/dataset?sort=extras_harvest_portal+asc%2C+score+desc%2C+metadata_modified+desc&q=state+boundary&organization=primeministerandcabinet&res_format=SHP) to go with that. But it's the current boundaries as opposed to boundaries over time (remember when Qld was part of NSW?)

**Comment by dicook** [comment]
The pedestrian sensors of Melbourne Open Data portal ([https://data.melbourne.vic.gov.au/Transport-Movement/Pedestrian-Counts/b2ak-trbp](https://data.melbourne.vic.gov.au/Transport-Movement/Pedestrian-Counts/b2ak-trbp)) could be a good one to get packaged in, if the JSON api can be utilized. 

And it would be good to make a different animation using R as an alternative to what is shown on the web site.

**Comment by jonocarroll** [comment]
@dicook sources like that would be very useful as 'cool things to plot on a map'. I have in mind something similar to the [LocationSA Map Viewer](http://location.sa.gov.au/viewer/) (though not necessarily interactive) with a variety of plot-ready data sets and a consistent mapping scheme.

[I'm working on a similar thing with vehicle traffic in Adelaide with leaflet/shiny myself](http://traffic.jcarroll.com.au/) and nearly have the point-to-point route code sorted so I can overlay routes taken (correlated bluetooth captures at traffic lights).

**Comment by dicook** [comment]
@jonocarroll are you suggesting that I list this as a separate topic? that it doesn't fall under what you were thinking for this issue?

**Comment by jonocarroll** [comment]
@dicook not at all, that repository looks to have some good data sets that could very well be included in my propsal, that pedestrian one in particular.

**Comment by dicook** [comment]
excellent! looking forward to this!

**Comment by Daniel-t** [comment]
Brisbane's Open Data (http://data.brisbane.qld.gov.au) program is getting underway and I know they are looking for suggestions for data to provide, so they will be interested in trying to provide anything city specific as well.   

There are some Brisbane based datasets available, such as:
- Ferry information - position, schedule etc (near realtime)
- Traffic Intersection data - flow rates (near realtime), unfortunately this is missing some meta data at the moment, such as which intersection is which
- Public art, park equipment and other static information.

(Disclaimer: I work for them)

-Daniel

**Comment by jonocarroll** [comment]
Looks like `raster::getData` already has the shapefile retrieval bit covered pretty well...

<img src="https://s3-ap-southeast-2.amazonaws.com/jcarroll1/auunconf/FRANCE.png">

<img src="https://s3-ap-southeast-2.amazonaws.com/jcarroll1/auunconf/AUSTRALIA.png">

Cheers for that, Adam!

**Comment by ivanhanigan** [comment]
Great to see these data, but where is the metadata a la https://github.com/ropensci/auunconf/issues/11 and https://github.com/ropensci/auunconf/issues/18?
looking forward to working on this in the near future.
